spec/models/facility_spec.rb

```diff
@@ -16,6 +16,11 @@
 RSpec.describe Facility, type: :model do
   let(:facility) { FactoryBot.create :facility }
 
+  it 'includes required modules' do
```

Comment:
These changes dont seem related to this PR. Would you mind making a separate PR for these changes that explains why these changes are needed?

Reply:
Will do it